kratos/containers/model.h

```diff
@@ -152,15 +151,76 @@ class KRATOS_API(KRATOS_CORE) Model final
      */
     std::vector<std::string> GetModelPartNames() const;
 
+   /**
+     * @brief Gets the value associated with a given variable.
+     * @tparam TDataType The data type of the variable.
+     * @param rThisVariable The variable for which the value is to be retrieved.
+     * @return Reference to the value of the provided variable.
+     */
+    template<class TDataType>
```

Comment:
the proposal of using the DataValueContainer as an object, was that instead of enrichning the interface of the model one would use the 

~~~py
model.GetDataValueContainer().GetValue(...)
~~~

this allows removing the functions from 154 to 190 as well as the Has method below

Reply:
Are you sure about that?, with this you will add more complexity to its usage. To set values in nodes or elements we do not call the container

Reply:
Just saying that we agreed this on the @KratosMultiphysics/technical-committee. The rationale is to avoid duplicating the interface of the `DataValueContainer` within the `Model`.